packages/Debian/arm64/DEBIAN/postinst

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+##--------------------------------------------------------------------
+## Copyright (c) 2018 Dianomic Systems
```

Comment:
Fix the copyright `2019 Dianomic Systems Inc.`

Reply:
Done